ci/run_locally/CI-build-docker-image.inputs

```diff
@@ -0,0 +1 @@
+chain=solo
```

Comment:
why only solo ?

Reply:
Because I should choose one of them, the use can change it if he want to build the other one.